### ISSUE TITLE
rclone rmdir doesn't remove directory

### ISSUE THREAD

**Issue by deliverer33** [ncw/rclone#956]
attempted 'rclone rmdir' on a acd mount.  The directory appears to delete, but after a period of time it appears again by running a ls command.  The directory is empty

rclone -V
rclone v1.34-67-g4428615β

**Comment by ncw** [comment]
Can you write a sequence of commands to Replicate please.

**Comment by deliverer33** [comment]
cd to acd.  Tried the following commands.

rm -rf "name"
rclone rmdir "name"

Both of these command show the directory gone after performing a 'ls.'  After a short while, when I run a "ls" command again, the previously deleted directory reappears.

**Comment by ncw** [comment]
I'm unable to replicate this.

Can you paste the command line you are using?

Can you cut and paste (or screenshot) a terminal session where you replicate the problem?

Thanks

Nick

**Comment by marrie** [comment]
Can you also do rclone delete remote:path/to/dir and see if that works? I also cannot reproduce this.

Thanks.

**Comment by Brandon168** [comment]
Same here. This behavior exists on Ubuntu and macOS Sierra for me. 

Reproduced using v1.35 on macOS Sierra. gdenc mount
[debug.txt](https://github.com/ncw/rclone/files/685420/debug.txt)
 is google drive using rclone crypt

`rclone mount gdenc:/ ~/gdenc > ~/debug.txt`

`rclone mkdir gdenc:/test`
`rclone mkdir gdenc:/test/dir1test`

* Delete dir1test using Finder
![screen shot 2017-01-04 at 11 22 23 am](https://cloud.githubusercontent.com/assets/923787/21651986/6af0fd44-d270-11e6-86d9-08446ee36167.png)
![screen shot 2017-01-04 at 11 22 28 am](https://cloud.githubusercontent.com/assets/923787/21651985/6aeef2e2-d270-11e6-8979-55ac840946fa.png)

* Wait a few minutes for good measure
* Folder still deleted per Finder view

`rclone lsd gdenc:/test/`
<img width="342" alt="screen shot 2017-01-04 at 11 26 23 am" src="https://cloud.githubusercontent.com/assets/923787/21652054/af0b9e9e-d270-11e6-966f-a1d3f37341a9.png">


Verbose output from mount attached
Note that unmounting and then remounting shows the "deleted" folder

I hope this is helpful

**Comment by Brandon168** [comment]
Note that the same behavior exists with non-encrypted google drive mount as well.

**Comment by ncw** [comment]
Ah, I see what happened I think....

If in your test you reverse the order in the setup

```
rclone mkdir gdenc:/test
rclone mkdir gdenc:/test/dir1test

rclone mount gdenc:/ ~/gdenc > ~/debug.txt
```

do you see the same effect?

**Comment by Brandon168** [comment]
Apologies, I don't understand the directions entirely. I should clarify that I used two different terminal windows for the original steps, one for the mount and another for the non-mount commands.

I followed the steps from your last comment:
```
rclone mkdir gdenc:/test
rclone mkdir gdenc:/test/dir1test
rclone mount gdenc:/ ~/gdenc > ~/debug.txt
```

* Use Finder to delete dir1test
* Open another terminal window and issue
`rclone lsd gdenc:/test`

The directory, dir1test, shows with that command. 

Of note, it does reappear in Finder as well after a while (close Finder .. wait a while, reopen the gdenc mount such that Finder refreshes.
